.circleci/config.yml

```diff
@@ -1,11 +1,16 @@
-version: 2.0
+version: 2.1
 
-jobs:
-  build-deploy: # build for the master branch
-    machine: true
-    working_directory: ~/addons-server
+commands:
+  build_and_push_container_image:
+    description: "Builds and pushes a Docker image"
+    parameters:
+      dockerfile:
+        type: string
+        default: "Dockerfile"
```

Comment:
It doesn't matter much because you always pass it explicitly, but default to `Dockerfile.py2.deploy` for consistency ?